style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
[Optional]
Kindly don't apply flex on the whole document. It'll make every child of HTML document go horizontal and center. You can apply flex to individual sections. 👍🏻